inc/block-patterns.php

```diff
@@ -20,9 +20,19 @@ function twentytwentytwo_register_block_patterns() {
 				'twentytwentytwo-query',
 				array( 'label' => __( 'Twenty Twenty-Two Posts', 'twentytwentytwo' ) )
 			);
+			register_block_pattern_category(
+				'twentytwentytwo-about-me',
+				array( 'label' => __( 'Twenty Twenty-Two: About me', 'twentytwentytwo' ) )
```

Comment:
I wonder if these need their own section, or could be included in the Page Layouts being added in #104. If their own section, nitpick on the capitalization: 
```suggestion
				array( 'label' => __( 'Twenty Twenty-Two: About Me', 'twentytwentytwo' ) )
```

Reply:
I think this is fine for now (with the capital M). We can move them over to Page Layouts after if we'd like.